impeller/renderer/backend/vulkan/swapchain/ahb/ahb_swapchain_impl_vk.cc

```diff
@@ -179,7 +184,7 @@ std::shared_ptr<ExternalFenceVK> AHBSwapchainImplVK::SubmitCompletionSignal(
 
   BarrierVK barrier;
   barrier.cmd_buffer = command_encoder_vk;
-  barrier.new_layout = vk::ImageLayout::eGeneral;
+  barrier.new_layout = vk::ImageLayout::ePresentSrcKHR;
```

Comment:
Did you mean to leave this in?

Reply:
I think once we fix the fd issue we should see if we even need this barrier? right now this is no-oping since we're already in general.

Reply:
Removed for now